vllm/config/compilation.py

```diff
@@ -564,3 +633,26 @@ def set_splitting_ops_for_v1(self):
     def splitting_ops_contain_attention(self) -> bool:
         return self.splitting_ops is not None and all(
             op in self.splitting_ops for op in self._attention_ops)
+
+    def get_compile_ranges(self) -> list[tuple[int, int]]:
+        """Get the compile ranges for the compilation config."""
+        compile_ranges_split_points = self.compile_ranges_split_points
```

Comment:
if a user specific 
compile_sizes = 16
and a range[1, 100, 1000]
we would split the ranges
to [1, 16], [16, 16] [16, 100]..

wonder if we want [16, 16]
and [1, 100] instead ..